src/main/webapp/app/modules/login/login.tsx

```diff
@@ -32,8 +34,11 @@ export class Login extends React.Component<ILoginProps, ILoginState> {
   };
 
   render() {
-    const { location, isAuthenticated } = this.props;
-    const { from } = location.state || { from: { pathname: '/', search: location.search } };
+    const { location, isAuthenticated, account } = this.props;
+    const defaultRoute = hasAnyAuthority(account.authorities, [AUTHORITIES.SACRAMENTO])
```

Comment:
I think you should rename it, it's not default role, it's default path

Reply:
it's 'route'